Lib/test/test_sys_settrace.py

```diff
@@ -833,9 +833,8 @@ def func():
              (5, 'line'),
              (6, 'line'),
              (7, 'line'),
-             (10, 'line'),
-             (13, 'line'),
-             (13, 'return')])
+             (10, 'line')] +
+             ([(13, 'line'), (13, 'return')] if __debug__ else [(10, 'return')]))
```

Comment:
The assertions on #819 affect the line numbers we are testing here when running with optimizations.